Formula/stm32flash.rb

```diff
@@ -0,0 +1,14 @@
+class Stm32flash < Formula
+  desc "Open source flash program for STM32 using the ST serial bootloader"
+  homepage "https://sourceforge.net/projects/stm32flash/"
+  url "https://downloads.sourceforge.net/project/stm32flash/stm32flash-0.5.tar.gz"
+  sha256 "97aa9422ef02e82f7da9039329e21a437decf972cb3919ad817f70ac9a49e306"
+
+  def install
+    system "make", "PREFIX=#{prefix}", "install"
+  end
+
+  test do
+    system "#{bin}/stm32flash", "-h"
```

Comment:
We need a test that exercises the some of the functionality of the app.

Reply:
Unfortunately there is nothing else you can do without a USB to UART converter and a compatible device connected.

Reply:
Can we try to perform an operation, and match the failure (exit code and error message)?